perf/specjbb/playlist.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+    <include>run/testenv.mk</include>
+    <test>
+        <!-- The minimum hardware requirement to run this test is 8GB of memory and 8 CPU cores -->
+        <!-- see specjbb/run/options/specjbb-multijvm-simple.sh for more details -->
+
+        <testCaseName>SPECjbb2015-multijvm-simple</testCaseName>
+        <command>. $(RUN_OPTIONS)/specjbb-multijvm-simple.sh; bash $(RUN_SCRIPTS)/run_multi.sh; $(TEST_STATUS)</command>
+        <levels>
+            <level>extended</level>
```

Comment:
```suggestion
            <level>dev</level>
```

Reply:
We can add this test to the dev level so it will never get included as a necessary test to run as part of AQAvit verification.